be/src/olap/wal/wal_manager.cpp

```diff
@@ -380,6 +386,8 @@ Status WalManager::_replay_background() {
         if (_first_replay.compare_exchange_strong(expected, false)) {
             RETURN_IF_ERROR(_load_wals());
         }
+        auto _total_wal_size = get_wal_queue_size(-1);
+        g_wal_total_count.set_value(_total_wal_size);
```

Comment:
can write to one line:
g_wal_total_count.set_value(get_wal_queue_size(-1));

Reply:
@mymeiyi already change